### PR TITLE
ARXIVNG-109 built out simple search

### PR DESCRIPTION
**Pull request by erickpeirson**

### CODE REVIEW

**Comment by erickpeirson** [comment]
I took this a bit further over the weekend, focusing on error messages and other UI bits. @eawoods eager to hear what you think of the changes to the advanced form, sort/page size, etc.

**Comment by eawoods** [comment]
Looking now - Erick, you're on fire! Given that you're traveling today, if I have adjustments for UI (not sure that I will, but asking just in case) - can I put them directly into a commit on the branch/PR? or would you prefer a comment and I can tidy up in a separate PR?

File: search/templates/search/author_search.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+              </div>
+              <div class="control">
+                <button type="button"
+                  data-toggle="fieldset-remove-row"
+                  id="authors-{{loop.index0}}-remove"
+                  class="button fieldset-hidden-on-first-row"
+                  aria-label="Remove this search term"
+                  {% if loop.index0 == 0 %}style="visibility:hidden;"{% endif %}
+                  >
+                    {# TODO: need something less hacky to get add/remove buttons the same width. #}
```
Comment (eawoods) [inline search/templates/search/author_search.html]: My suggestion for the less-hacky method: how about I specify a class "is-monospace"? If we use the Lucida Grande font for buttons, the + and - symbols are the same width. We can also apply width-controlling CSS parameters on buttons of a group in this way.

**Comment by mhl10** [comment]
We're going to merge now this to keep things moving

File: search/templates/search/author_search.html (hunk 1 above)
Comment (erickpeirson) [reply search/templates/search/author_search.html]: That sounds excellent

**Comment by eawoods** [comment]
I didn't say this yet, cause I didn't see it till now - beautiful job on the error handling and design for advanced search page. :)

**Comment by erickpeirson** [comment]
@eawoods Thanks!

Re macros: actually, some of those are orphan; I duped the advanced_search.html template and failed to clean up.

- select_field() isn't used in authors_search.html
- advanced_query() should be renamed to author_query()
- size_and_order() should definitely be moved to search-macros.html

Just noticed that some of our template filenames use underscore, others use hyphens. Should we pick one?

**Comment by eawoods** [comment]
Underscore versus hyphen: yeah, and that's probably me. CSS classes use hyphens, everything else (it seems) uses underscores.....I'm agnostic about file names and happy to go along with either if there's a preference.

**Comment by eawoods** [comment]
@erickpeirson I can clean up the macros. Also, I didn't want to bug you at ACM - note the list of issues I'm logging to collect tasks/bugs/improvements, to be assigned and addressed as the team sees fit.

**Pull request merged**